.github/workflows/license-checker.yml

```diff
@@ -3,10 +3,10 @@ name: License checker
 on:
   push:
     branches:
-      - master
+      - planner_refactory
   pull_request:
     branches:
-      - master
+      - planner_refactory
```

Comment:
After the protection of planner_refactory is turned off, I will restore the changes here.